live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/e2e/UpdateRequestIntegrationTest.java

```diff
@@ -124,4 +124,244 @@ void testUpdateRequest()
     }
+
+    @Test
+    void shouldCreateCustomProperty()
+    {
+        // given
+        containerSupport.prepareHxInsightToReturnSuccess();
+
+        String properties = """
+                {
+                  "cm:title": "Purchase Order"
+                }
+                """;
+
+        String propertiesBefore = """
+                {
+                  "cm:title": null
+                }
+                """;
+
+        String repoEvent = generateCustomPropertiesUpdatedEvent(properties, propertiesBefore);
+
+        // when
+        containerSupport.raiseRepoEvent(repoEvent);
+
+        // then
+        String expectedBody = """
+                {
+                  "objectId" : "d71dd823-82c7-477c-8490-04cb0e826e65",
+                  "eventType" : "update",
+                  "properties" : {
+                    "cm:title": "Purchase Order"
+                  }
+                }""";
+        containerSupport.expectHxIngestMessageReceived(expectedBody);
+    }
+
+    @Test
+    void shouldUpdateCustomProperty()
+    {
+        // given
+        containerSupport.prepareHxInsightToReturnSuccess();
+
+        String properties = """
+                {
+                  "cm:title": "Summary for year 2024"
+                }
+                """;
+
+        String propertiesBefore = """
+                {
+                  "cm:title": "Summary"
+                }
+                """;
+
+        String repoEvent = generateCustomPropertiesUpdatedEvent(properties, propertiesBefore);
+
+        // when
+        containerSupport.raiseRepoEvent(repoEvent);
+
+        // then
+        String expectedBody = """
+                {
+                  "objectId" : "d71dd823-82c7-477c-8490-04cb0e826e65",
+                  "eventType" : "update",
+                  "properties" : {
+                    "cm:title": "Summary for year 2024"
+                  }
+                }""";
+        containerSupport.expectHxIngestMessageReceived(expectedBody);
+    }
+
+    /**
+     * This weird situation when we update property from null -> null sometimes happens, and it is probably some defect of our event system
+     */
+    @Test
+    void shouldDoNothingWithUnchangedProperty()
+    {
+        // given
+        containerSupport.prepareHxInsightToReturnSuccess();
+
+        String properties = """
+                {
+                  "cm:taggable": null
+                }
+                """;
+
+        String propertiesBefore = """
+                {
+                  "cm:taggable": null
+                }
+                """;
+
+        String repoEvent = generateCustomPropertiesUpdatedEvent(properties, propertiesBefore);
+
+        // when
+        containerSupport.raiseRepoEvent(repoEvent);
+
+        // then
+        String expectedBody = """
+                {
+                  "objectId" : "d71dd823-82c7-477c-8490-04cb0e826e65",
+                  "eventType" : "update"
+                }""";
```

Comment:
Hmm - this is a weird edge case. I guess really we shouldn't send anything in this case, but actually it should never happen since I think the modification date will always be updated.

Reply:
I can actually add some random property like title updated wdyt?

Reply:
so we'll have 
![image](https://github.com/Alfresco/hxinsight-connector/assets/62990104/ded2222a-ea8c-457a-ad62-684bb2435df3)

Reply:
I'm happy either way. Like this we at least check what happens when there's no real change, even if it's a bit weird to tell Hx Insight "there was no change to this node".